tox.ini

```diff
@@ -58,7 +59,7 @@ deps =
   docs: setuptools >= 36.4.0
   docs: sphinx >= 1.6.3
 
-commands = 
+commands =
   py{27,34,35}-unit: {[constants]unit-base-command} --ignore=contrib/opencensus-ext-stackdriver --ignore=contrib/opencensus-ext-flask --ignore=contrib/opencensus-ext-httpx
```

Comment:
Since `fastapi` is not supported for python < 3.7, you can add `--ignore=contrib/opencensus-ext-fastapi` to make the builds pass.

Reply:
https://github.com/census-instrumentation/opencensus-python/pull/1124#:~:text=on%20python%20%3C%203.7-,4ec02b7,-Merge%20state